projects/storefrontlib/src/cms-components/navigation/page-header/page-header.component.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { I18nTestingModule, PageMeta, PageMetaService } from '@spartacus/core';
+import { CmsComponentData } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { PageHeaderComponent } from './page-header.component';
+
+class MockPageMetaService {
+  getMeta(): Observable<PageMeta> {
+    return of(<PageMeta>{
+      title: 'Test title',
+      description: 'Test description',
+    });
+  }
+}
+
+describe('PageHeaderComponent', () => {
+  let component: PageHeaderComponent;
+  let fixture: ComponentFixture<PageHeaderComponent>;
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [RouterTestingModule, I18nTestingModule],
+        declarations: [PageHeaderComponent],
+        providers: [
+          { provide: PageMetaService, useClass: MockPageMetaService },
+          {
+            provide: CmsComponentData,
+            useValue: {
+              data$: of({}),
+            },
+          },
+        ],
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PageHeaderComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
There are no tests in this spec.  Please add coverage.

Reply:
TBH I didn't add any new logic here, I only divided BreadCrumbComponent to two smaller components. But sure, I can add it

Reply:
Ah I understand!  It didn't have coverage to begin with.  Since it's separated out in it's own component, if you can add some coverage, it would be great.